### PR TITLE
feat: SIGINT-852 Airgap and coverity newer version support

### PR DESCRIPTION
**Pull request by kiransynopsys**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -220,10 +221,11 @@ Pass the following additional parameters as necessary.
...
 | `INCLUDE_DIAGNOSTICS`      | Synopsys Bridge diagnostics files will be available to download when it is set to `true`.<br/> Azure DevOps no longer supports per-pipeline retention rules. The only way to configure retention policies for YAML and classic pipelines is through the project settings.<br/> Refer the given documentation for more details: <br/> https://learn.microsoft.com/en-us/azure/devops/pipelines/policies/retention?view=azure-devops&tabs=yaml#set-run-retention-policies               | Optional     |
+| `NETWORK_AIR_GAP` | If the `NETWORK_AIR_GAP` is set to true, Synopsys Security Scan Extension for Azure DevOps will not download the Synopsys Bridge but instead uses the pre-configured Synopsys Bridge. If the Synopsys Bridge is configured at a specific location, provide the path through `SYNOPSYS_BRIDGE_INSTALL_DIRECTORY`, <br/><br/>The Synopsys Security Scan Extension will look for the Synopsys Bridge from `SYNOPSYS_BRIDGE_INSTALL_DIRECTORY` path otherwise it will look for the Synopsys Bridge in the default path ($HOME/synopsys-bridge). | Optional     |
```
Comment (sig-tithi) [inline README.md:228]: is it bridge_network_airgap?

File: README.md [hunk 2]
```diff
@@ -33,8 +33,8 @@ Before configuring Synopsys Security Scan into your azure pipeline, note the fol
...
 
 - `AZURE_TOKEN` is required as input when running Black Duck Fix PR, Black Duck/Coverity PR Comment. There are 2 different types of tokens that can be passed to `AZURE_TOKEN`
-  1. When using `AZURE_TOKEN: $(System.AccessToken)`, you must enable this in the Azure interface. Go to Project → Project Settings → Repository → Security → Build Service and set `Contribute to pull requests` to `Allow`. <br> Confirm `System.AccessToken` has Contribute to PR permissions (Project → Project Settings → Repositories → Security → Build Service User)
-  2. When using `AZURE_TOKEN: $(PAT_TOKEN)`, PAT token should have minimum permissions `Code - Full` and `Pull Request Threads - Read & write`. Refer [Use personal access tokens](https://learn.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate?view=azure-devops&tabs=Windows) for more details.
+  1. When using `AZURE_TOKEN: $(System.AccessToken)`, confirm it has required permissions to create Pull Requests. Go to Project → Project Settings → Repository → Security → Build Service User and set `Contribute to pull requests`, `Create branch` and `Delete or disable repository` to `Allow`.
```
Comment (spurohitsynopsys) [inline README.md:36]: these are not un sync with main branch

Comment (lsynopsys) [reply README.md:36]: it is not only for create pull request also for branch creation as part of fix pr

**Review by spurohitsynopsys — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (sig-tithi) [reply README.md]: i reverted to what was there in main as discused.

**Pull request merged**